content/documentation/commands.md

```diff
@@ -0,0 +1,105 @@
++++
+title = "Commands"
+weight = 21
++++
+
+## build (Build the project)
```

Comment:
Suggestion for all titles:

I don't like `build (Build the project)`

I think makes more sense something like `Build the project (phel build)` or simply `Build the project`, then in the description we are declaring the exact command the user should run instead.

Reply:
I thought the same, and I wasn't sure about this. But now that you mentioned it, I will change it as you propose: `simply "Build the project"` 👍🏼